app/views/auth/login.blade.php

```diff
@@ -6,7 +6,7 @@
             <div class="welcome-logo">
                 <img class="logo" height="50" src="{{ url('img/cachet-logo.svg') }}" alt="Cachet"/>
             </div>
-            {{ Form::open() }}
+            {{ Form::open(['url' => '/auth/login']) }}
```

Comment:
I prefer using named routes, if we have them defined I think it's handy for changes on the URI's also.